au/au.hh

```diff
@@ -34,4 +34,14 @@ struct CorrespondingQuantity<std::chrono::duration<RepT, Period>> {
     static constexpr ChronoDuration construct_from_value(Rep x) { return ChronoDuration{x}; }
 };
 
+// Convert any Au duration quantity to an equivalent `std::chrono::duration`.
```

Comment:
Q: au.hh's two functions seem to be `std::chrono` related. Is it worth breaking that out into its own files?

Reply:
That's a good point.  I'll do that in a follow-up PR.

Reply:
#201.